src/Authentication/Authentication.Core/Utilities/AuthenticationHelpers.cs

```diff
@@ -129,16 +129,13 @@ private static async Task<InteractiveBrowserCredential> GetInteractiveBrowserCre
                     authRecord = await Task.Run(() =>
                     {
                         // Run the thread in MTA.
+                        GraphSession.Instance.OutputWriter.WriteWarning("Note: Signin by Web Account Manager (WAM) is enabled by default on Windows. If using an embedded PowerShell terminal, the interactive browser window may be hidden behind other windows.");
                         return interactiveBrowserCredential.Authenticate(new TokenRequestContext(authContext.Scopes), cancellationToken);
                     });
                 }
                 else
                 {
-                    authRecord = await Task.Run(() =>
```

Comment:
Why was the MTA run removed?

Reply:
A better question is why was an async call wrapped in a Task.Run in the first place? 

The `AuthenticateAsync` method being called here returns `Task<AuthenticationRecord>` where as the `Authenticate` method above is synchronous and needs to wrapped to allow it to run on a separate thread and be awaited.